lifecycle.go

```diff
@@ -475,13 +475,13 @@ func (p *DockerProvider) preCreateContainerHook(ctx context.Context, req Contain
 	return nil
 }
 
-// combineContainerHooks it returns just one ContainerLifecycle hook, as the result of combining
+// CombineContainerHooks it returns just one ContainerLifecycle hook, as the result of combining
```

Comment:
Same here: we don't want users to combine the hooks, as this is internal to the project. Which makes me think we could move this to the internal package 🤔

Reply:
This one could probably be moved yeah, I did a thorough search and none of the tests using it seem to be loaded as examples for the docs 🤔